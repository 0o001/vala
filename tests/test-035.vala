using GLib;

class SwitchTest {
    static void main () {
        var foo = "Hello World";
        var bar = 0;

        stdout.printf ("before switch: %s\n", foo);

	switch (bar) {
	    case 0:
		stdout.printf ("within switch: %s\n", foo);
		break;
	}

        stdout.printf ("behind switch: %s\n", foo);
    }
}
