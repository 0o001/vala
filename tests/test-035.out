before switch: Hello World
within switch: Hello World
behind switch: Hello World
